docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+function which will be the entry point of our executable program:
+
+.. code-block:: js
+
+    package main
+
+    import (
+        "encoding/json"
+        "fmt"
+        "io/ioutil"
+        "log"
+        "net/http"
+    )
+
+    func main() {
+
+    }
+
+Then, look at the JSON data that gets returned from going to the Open
+Notify API endpoint at `<http://api.open-notify.org/iss-now.json>`_. The 
+endpoint returns a JSON payload, which contains an ``iss_position`` object
+with ``latitude`` and ``longitude`` data.
+
+.. code-block:: js
+
+   {
+        "message":"success",
+        "timestamp":1591703638,
+        "iss_position":{
+            "longitude":"84.9504",
+            "latitude":"41.6582"
+        }
+    }
+
+The longitude and latitude of the International Space Station changes 
+constantly and is what you want to extract from this payload and insert into
+CrateDB. To parse this JSON, you can create a `struct`_ that can be used to 
+`unmarshal`_ the data into. When you unmarshal JSON into a struct, the 
+function matches incoming object keys to the keys in the struct field name
+or its tag. By default, object keys which don't have a corresponding struct 
+field are ignored.
+
+.. code-block:: js
+
+    type issInfo struct {
+        IssPosition struct {
+            Longitude string `json:"longitude"`
+            Latitude  string `json:"latitude"`
+        } `json:"iss_position"`
+    }
+
+Now, create a function that makes an HTTP GET request to the Open Notify API 
+endpoint and returns longitude and latitude as a `geo_point`_ declaration.
+
+The Go standard library comes with great support for HTTP client and server
+implementations in the `net/http`_ package. In this example you’ll use it to 
+issue an HTTP GET request to the API endpoint. 
+
+Then you will read the response body and unmarshal the JSON into your defined 
+struct ``issInfo``. 
+
+Lastly, you will format the return string and return it. 
+
+.. code-block:: js
+
+    func getISSPosition() (string, error) {
+        var i issInfo
+
+        response, err := http.Get("http://api.open-notify.org/iss-now.json")
+        if err != nil {
+            return "", fmt.Errorf("unable to retrieve request: %v", err)
+        }
+        defer response.Body.Close()
+
+        if response.StatusCode/100 != 2 {
+            return "", fmt.Errorf("bad response status: %s", response.Status)
+        }
+
+        responseData, err := ioutil.ReadAll(response.Body)
+        if err != nil {
+            return "", fmt.Errorf("unable to read response body: %v", err)
+        }
+
+        err = json.Unmarshal(responseData, &i)
+        if err != nil {
+            return "", fmt.Errorf("unable to unmarshal response body: %v", err)
+        }
+
+        s := fmt.Sprintf("(%s, %s)", i.IssPosition.Longitude, i.IssPosition.Latitude)
+        return s, nil
+    }
+
+In your main function, call your ``getISSPosition()`` function and print out the result:
```

Comment:
would use "the" instead of "your" here. "you", "your" for talking about things the reader should do, has done, etc, but the definite article ("the") for data/code we are sharing with the reader